backend/internal/storage/postgres/schema/media.go

```diff
@@ -311,16 +311,16 @@ func (r *MediaRepository) GetMediaByReviews(ctx context.Context, limit, offset i
 		a.id AS album_id,
 		a.title AS album_title,
 		a.release_date,
-		a.cover,
+		a.cover as cover,
 		a.country,
 		a.genre_id,
 		t.id AS track_id,
 		t.title AS track_title,
 		t.album_id AS track_album_id,
 		t.duration_seconds
 	FROM MostReviewed m
-	LEFT JOIN album a ON m.media_id = a.id AND m.media_type = 'album'
 	LEFT JOIN track t ON m.media_id = t.id AND m.media_type = 'track'
+	JOIN album a ON (t.album_id = a.id OR (m.media_id = a.id AND m.media_type = 'album'))
```

Comment:
changed this join so that track covers get populated via their respective album covers (without it, the frontend sometimes had no image to show for tracks)